### DISCUSSION TITLE
My Achievements not show

### DISCUSSION THREAD

**Discussion by seon22break** [community/community#45125] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Due to a bug, it seems that the Achievements are not displayed in my profile. I am certain that I have achieved more than one...

**Comment by jge162** [comment, marked as answer]
which achievement are you missing? Maybe you did not complete it. what steps did you take?

**Reply by seon22break** [reply]
For example : Pair Extraordinaire or Quickdraw ?

**Reply by jge162** [reply]
for quick draw what did you do? to verify you did it right

**Reply by jge162** [reply]
you need to open a issue in one of your repositories, once up close the issue within 5 mins. did u do that? @seon22break

**Reply by seon22break** [reply]
In Extraordinaire Quickdraw, I am coauthored in some pull requests and merge into master. And in Quickdraw not sure : Repository has been public?

**Reply by jge162** [reply]
@seon22break public yes for quickdraw

coauthored, but you need to create a new fork using GitHub desktop app for (windows or mac)

**Reply by seon22break** [reply]
Ok! Thanks for your help. I dont create fork with Desktop app.  I has proved obtain Yolo Archivement and work fine! Sorry

**Reply by rxl895** [reply]
I have also contributed to my public repo, should obtain more than 2 achievements, but gotten none.
